<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FpgaUpdatePageClass</class>
 <widget class="QWidget" name="FpgaUpdatePageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FpgaUpdatePage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="te_log"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="hl_target_chip">
       <item>
        <widget class="QLabel" name="lbl_target_chip">
         <property name="text">
          <string>目标芯片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_target_chip"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_flashing_mode">
       <item>
        <widget class="QLabel" name="lbl_flashing_mode">
         <property name="text">
          <string>烧写模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_flashing_mode"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_probes">
       <item>
        <widget class="QLabel" name="lbl_probes">
         <property name="text">
          <string>烧录器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_probes"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_scan_probes">
         <property name="text">
          <string>扫描</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_device_id">
       <item>
        <widget class="QLabel" name="lbl_devices">
         <property name="text">
          <string>设备ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_devices"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_scan_devices">
         <property name="text">
          <string>扫描</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_firmware_path">
       <item>
        <widget class="QLabel" name="lbl_firmware_path">
         <property name="text">
          <string>固件路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_firmware_path"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_select_fiirmware">
         <property name="text">
          <string>选择固件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_wirte_firmware">
         <property name="text">
          <string>烧录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_programming">
       <item>
        <widget class="QLabel" name="lbl_programming">
         <property name="text">
          <string>烧写进度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressbar_programming">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_verifying">
       <item>
        <widget class="QLabel" name="lbl_verifying">
         <property name="text">
          <string>校验进度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressbar_verifying">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
